xtext-website/documentation/310_eclipse_support.md

```diff
@@ -514,6 +514,34 @@ Often the default strategy only needs some guidance (e.g. selecting the text cor
 
 The hyperlinks are provided by the [HyperlinkHelper]({{site.src.xtext_eclipse}}/org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hyperlinking/HyperlinkHelper.java) which will create links for cross-referenced objects by default. Clients may want to override `createHyperlinksByOffset(XtextResource, int, IHyperlinkAcceptor)` to provide additional links or supersede the default implementation.
 
+## Hovering {#hovering}
+Similar to [hyperlinking](#hyperlinking), Xtext-based editors provide hovering support on tokens corresponding to cross-references out-of-the-box. When hovering over a cross-reference token, the Xtext framework shows the documentation of the element the cross-reference is referring to. Considering the Xtext Simple Arithmetics example, when hovering over e.g. a function call, a popup window displays the documentation of the called function:
```

Comment:
actually you con hover at everything that represents a name. (declarations + references)

Reply:
done